xml/abstract_commands.xml

```diff
@@ -197,25 +219,39 @@
             physical accesses, but it must fail accesses that it doesn't
             support.
 
-            0: Addresses are physical (to the hart they are performed on).
+            <value v="0" name="physical">
+            Addresses are physical (to the hart they are performed on).
+            </value>
 
-            1: Addresses are virtual, and translated the way they would be from
+            <value v="1" name="virtual">
+            Addresses are virtual, and translated the way they would be from
             M-mode, with \FcsrMstatusMprv set.
+            </value>
 
             Debug Modules on systems without address translation (i.e. virtual addresses equal physical)
             may optionally allow \FacAccessmemoryAamvirtual set to 1, which would produce the same result as
             that same abstract command with \FacAccessmemoryAamvirtual cleared.
         </field>
         <field name="aamsize" bits="22:20">
-            0: Access the lowest 8 bits of the memory location.
+            <value v="0" name="8">
```

Comment:
Similar concern about the name being just a number.

Reply:
Similar response.

Reply:
Same here.

Reply:
Fixed.